latex/epoch.tex

```diff
@@ -123,48 +122,82 @@ \subsection{Stake Distribution Calculation}
 These stake-related calculations are used in the calculation of rewards.
 
 %%
-%% Figure - Functions for Stake Distribution
+%% Figure - Helper Functions for Stake Distribution
 %%
 \begin{figure}[htb]
+  \emph{Stake Distribution helper functions}
+  %
   \begin{align*}
-      & \fun{baseStake} \in \powerset{\TxOut} \to \powerset{(\HashKey \times \Coin)}
-      & \text{base address stake} \\
-      & \fun{baseStake}~{outs} = \\
-      & ~~ \{(hk, c) \mid a\in\Addr_{base} \land (a, c) \in outs \land \fun{stakeHK}~a = hk \}
+      & \fun{consolidate} \in \UTxO \to (\Addr \mapsto \Coin) \\
+      & \fun{consolidate}~{utxo} =
+        \left\{a \mapsto \left(\sum_{\wcard\mapsto(a,c)\in\var{utxo}}c\right)
+        ~\Big\vert~
+        \wcard\mapsto(a,~\wcard)\in\var{utxo} \right\} \\ 
       \nextdef
-      & \fun{ptrStake} \in \powerset{\TxOut} \to (\Ptr \mapsto \HashKey)
-          \to \powerset{(\HashKey \times \Coin)}
-      & \text{pointer address stake} \\
-      & \fun{ptrStake}~{outs}~{ptrs} = \\
-      & ~~ \{(hk, c) \mid a\in\Addr_{ptr} \land (a, c) \in outs \land (\fun{stakePtr}~a)\mapsto hk \in ptrs \}
+      & \fun{baseStake} \in (\Addr\mapsto\Coin) \to \Stake \\
+      & \fun{baseStake}~{outs} =
```

Comment:
maybe rename the `outs` parameter to something else as it is not a set of `TxOut` any more, same for `ptrStake`

Reply:
yea, I was having a hard time coming up with a good name...